src/stylesheets/app.scss

```diff
@@ -0,0 +1,298 @@
+@import "base/vars";
+@import "base/typography";
+
+body {
+  background-color: $florence-light-blue; // only temporary
+  margin: 0;
+}
+
+ul {
+  padding: 0;
+}
+
+sup {
+  color: #e95e67;
+  text-decoration: underline;
+}
+
+/*-------- Top Fold styles ---------*/
```

Comment:
To keep this file from getting too long, you could break it apart into separate "section" sass files, so, **sections/top.scss**, **sections/pillbar.scss**, **sections/adherence.scss**, etc.